modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,41 @@
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
+
+Network Observability requires a substantial amount of resources. There are settings to consider that can help you get an optimal setup without compromising on the observability.
+
+The following settings can help you manage resources and performance:
+
+    Resource requirements and limits:: Adapt the resource requirements and limits to the load and memory usage you expect on your cluster by using the `spec.agent.ebpf.resources` and `spec.processor.resources` specifications. The default limits (800MB) should be sufficient for most medium sized clusters.
+    
+    Cache max flows timeout:: Control how often flows are reported by the agents by using the eBPF agent's `spec.agent.ebpf.cacheMaxFlows` and `spec.agent.ebpf.cacheActiveTimeout` specifications. The higher the value specified, the less traffic is generated by the agents, which correlates with less CPU load. However, it leads to a slightly higher memory consumption, and might generate more latency in the flow collection.
+
+    Restricting or excluding interfaces::  Reduce the overall observed traffic using `spec.agent.ebpf.interfaces` and `spec.agent.ebpf.excludeInterfaces`. Note that the interface names might vary according to the CNI used.
+
+The following table documents recommendations for resources depending on workload size. 
+[IMPORTANT]
+====
+These recommended scenarios are tailored to specific workloads and should serve only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Resource recommendations
+[options="header"]
+|============================================================================================================================
+|                                     | Extra small (10 Nodes) | Small (25 nodes)  | Medium (65 Nodes)  | Large (125 nodes) 
+| *Worker Node Size*                  | m5.xlarge (AWS)        | m5.4xlarge (AWS)  | m5.4xlarge (AWS)   | m5.4xlarge (AWS)
+>| vCPU                                >| 4 vCPUs             >| 16 vCPUs         >| 16 vCPUs          >| 16 vCPUs
+>| Mem                                 >| 16GiB Mem           >| 64GiB Mem        >| 64GiB Mem         >| 64GiB Mem
+| *LokiStack Size*                    | 1x.extra-small         | 1x.small          | 1x.small           | 1x.medium                  
+| *NetObserv Controller Memory Limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 800Mi                
+| *eBPF Sampling Rate*                | 50 (default)           | 50 (default)      | 50 (default)       | 50(default)
+| *eBPF Memory Limit*                 | 800Mi (default)        | 800Mi (default)   | 2000Mi             | 800Mi (default) 
+| *FLP CPU Limit*                     | None                   | None              | None               | None                
```

Comment:
Wondering if we should just eliminate this row entirely?

Reply:
Ah yeah good point since they are defaults maybe they're unnecessary. I can remove them.

Reply:
For the record I was referring to `FLP CPU Limit` specifically, I think the others could be kept

Reply:
Thanks for clarifying, yeah for some reason the suggestion makes it seem like lines 32-35, which are all repetitive so I thought you meant all. But OK, just FLP CPU limit it is.